scripts/compose/docker-compose.yaml

```diff
@@ -77,12 +74,33 @@ services:
     ports:
       - '9093:8080'
     networks:
-      - hopr-nat
+      - hopr-net
     devices:
       - /dev/kmsg:/dev/kmsg
 
+  nodeexporter:
```

Comment:
This will likely have limited usability in cloud deployments though, e.g. helm charts should likely not include this the node exporter.

Reply:
This is not intended for cloud deployments, as they have their configurations in a separate repo.